platform/local_country_file.hpp

```diff
@@ -69,6 +69,7 @@ class LocalCountryFile
   bool operator<(LocalCountryFile const & rhs) const;
   bool operator==(LocalCountryFile const & rhs) const;
 
+  void SetVersionForTesting(int64_t version) { m_version = version; }
```

Comment:
What the point in this method? Especially in presence of constructor of this class (it takes version as an argument) and LocalCountryFile::MakeForTesting?